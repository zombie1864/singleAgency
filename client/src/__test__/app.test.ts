import {render,fireEvent} from '@testing-library/react'

test('First test', () => {
    expect(true).toBeTruthy()
})

export default test 