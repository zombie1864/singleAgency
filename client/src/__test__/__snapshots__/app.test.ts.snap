// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing async action creator dispatch setData on succesful api call 1`] = `
Array [
  Object {
    "fixture": undefined,
    "type": "SET_DATA",
  },
]
`;
